src/VCR_InterfaceTasks.cpp

```diff
@@ -15,52 +15,68 @@
+
+    vcr_data.interface_data.current_sensor_data.current_sensor_unfiltered = 
+        ADCSingletonInstance::instance().adc0.data.conversions[CURRENT_SENSE_CHANNEL].conversion;
+
+    vcr_data.interface_data.current_sensor_data.current_refererence_unfiltered = 
+        ADCSingletonInstance::instance().adc0.data.conversions[REFERENCE_SENSE_CHANNEL].conversion;
+
+    vcr_data.interface_data.rear_loadcell_data.RL_loadcell_analog = 
+        ADCSingletonInstance::instance().adc0.data.conversions[RL_LOADCELL_CHANNEL].conversion;
+
+    vcr_data.interface_data.rear_loadcell_data.RR_loadcell_analog = 
+        ADCSingletonInstance::instance().adc0.data.conversions[RR_LOADCELL_CHANNEL].conversion;
+
+    vcr_data.interface_data.rear_suspot_data.RL_sus_pot_analog = 
+        ADCSingletonInstance::instance().adc0.data.conversions[RL_SUS_POT_CHANNEL].raw; // Just use raw for suspots
+
+    vcr_data.interface_data.rear_suspot_data.RR_sus_pot_analog = 
+        ADCSingletonInstance::instance().adc0.data.conversions[RR_SUS_POT_CHANNEL].raw; // Just use raw for suspots
 
-    vcr_data.interface_data.current_sensor_data.twentyfour_volt_sensor = ADC0Instance::instance().data.conversions[GLV_SENSE_CHANNEL].conversion;
-    vcr_data.interface_data.current_sensor_data.current_sensor_unfiltered = ADC0Instance::instance().data.conversions[CURRENT_SENSE_CHANNEL].conversion;
-    vcr_data.interface_data.current_sensor_data.current_refererence_unfiltered = ADC0Instance::instance().data.conversions[REFERENCE_SENSE_CHANNEL].conversion;
-    vcr_data.interface_data.rear_loadcell_data.RL_loadcell_analog = ADC0Instance::instance().data.conversions[RL_LOADCELL_CHANNEL].conversion;
-    vcr_data.interface_data.rear_loadcell_data.RR_loadcell_analog = ADC0Instance::instance().data.conversions[RR_LOADCELL_CHANNEL].conversion;
-    vcr_data.interface_data.rear_suspot_data.RL_sus_pot_analog = ADC0Instance::instance().data.conversions[RL_SUS_POT_CHANNEL].raw; // Just use raw for suspots
-    vcr_data.interface_data.rear_suspot_data.RR_sus_pot_analog = ADC0Instance::instance().data.conversions[RR_SUS_POT_CHANNEL].raw; // Just use raw for suspots
     // Serial.println("yo");
-    hal_printf("ADC0 reading 0 %d\n", ADC0Instance::instance().data.conversions[0].raw); // NOLINT
+    hal_printf("ADC0 reading 0 %d\n", 
+        ADCSingletonInstance::instance().adc0.data.conversions[0].raw); // NOLINT
 }
 
 bool init_read_adc1_task()
 {
-    /* NOLINTBEGIN */ // Thermistor channels are for testing purposes only, the pin numbers 0-7 are acceptable "magic numbers".
-    // Initialize all eight channels to scale = 1, offset = 0
-    float scales[channels_within_mcp_adc] = {1, 1, 1, 1, 1, 1, 1, 1};
-    float offsets[channels_within_mcp_adc] = {0, 0, 0, 0, 0, 0, 0, 0};
-    /* NOLINTEND */
-
-    ADC1Instance::create(ADC1_CS, MCP_ADC_DEFAULT_SPI_SDI, MCP_ADC_DEFAULT_SPI_SDO, MCP_ADC_DEFAULT_SPI_CLK, MCP_ADC_DEFAULT_SPI_SPEED, scales, offsets);
-
+    init_bundle();
```

Comment:
why are we init-ing bundle twice? You shouldn't be calling the create() function again on a singleton instance.